oat-sa/extension-tao-proctoring
# Feature/actp 342/move column to primary table

JIRA: https://oat-sa.atlassian.net/browse/ACTP-342

The goal of this development is to improve performance on proctoring screen. For that reason we move some fields common for all instances with proctoring to main `delivery_monitoring` table. With this change it won't be necessary to JOIN `kv_delievry_monitoring` table to get sessions for proctor screen. 
Add `GROUP BY` statement in query only when we join `kv_delivery_monitoring` to improve performance.

**How to test:**
- Fresh install - `test_taker_first_name, test_taker_last_name, remaining_time, extended_time, extra_time, consumed_extra_time, last_test_taker_activity` columns must should be present in delivery_monitoring table. 
- Update - `test_taker_first_name, test_taker_last_name, remaining_time, extended_time, extra_time, consumed_extra_time, last_test_taker_activity` columns created in `delivery_monitoring` table and all existing records for these keys migrated to `delivery_monitoring`. Config `config/taoProctoring/DeliveryMonitoring.conf.php` is updated with new primary columns.

In both cases `kv_delivery_monitoring` is not used to get data for proctor screen.
